packages/beacon-node/test/spec/bls/bls.ts

```diff
@@ -44,7 +42,7 @@ function aggregate_verify(input: {pubkeys: string[]; messages: string[]; signatu
       pubkeys.map((pk) => PublicKey.fromHex(pk)),
       Signature.fromHex(signature)
     );
-  } catch (e) {
```

Comment:
it might be useful to have the error for debugging purposes, maybe we can rename `_e` to fix lint error

Reply:
If we need and use it, we can add it then.

Reply:
if you have a running process and attach a debugger you can't just change it on the fly

Reply:
I usually don't debug a process on the fly, if I had to debug a specific code and specific code flow, will work on it. 

But if everyone else feels the same we can keep add unused variables for exceptions. Doing it in a separate PR will much ideal to track for future that we explicitly added. it.

Reply:
I agree with @nflaig we should avoid code updates in this PR, please rename error with underscore on unused error variables.